### DISCUSSION TITLE
UI translations?

### DISCUSSION THREAD

**Discussion by zblace** [pretalx/pretalx#1404] [Q&A] [answered]
Is there a list of UI languages and how far are they supported *(guessing only EN and DE fully)?
How can one check and add new UI language translations?

**Comment by rixx** [comment, marked as answer]
You can look at (and contribute to!) translations at [translate.pretalx.com](https://translate.pretalx.com/projects/pretalx/pretalx/#translations).

**Reply by zblace** [reply]
It is a pity that there is still no other language than English at 100% and that very close are only Czech and Portuguese. 
Have you considered commissioning at least to finish few more?

**Reply by rixx** [reply]
German is also functionally at 100% (always updated for full releases). Anything above 80% is typically functional, so French, Spanish, Arabic and Japanese should also work. Eight languages is a pretty decent translation statistic for a project of this size!

pretalx is open source software and relies on the community to contribute features they desire. While I'm the main (and often only) developer, I don't make a living off pretalx, and commissioning translations is an ongoing financial investment that I cannot make. (Ongoing because text is added and modified all the time, which is why languages that used to be at 100% sit now a bit below that.) Contributions at the linked platform are always welcome.

**Reply by zblace** [reply]
Understandable...Thank you for elaboration! Will consider it for the bigger events I organize in the future.
